components/app.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import LocationDetails from './location-details';
+import ForecastSummaries from './forecast-summaries';
+import ForecastDetails from './forecast-detils';
+import SearchForm from './search-form';
+
+import '../src/styles/app.scss';
+
+const URL = 'https://mcr-codes-weather.herokuapp.com/forecast?city=';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedDate: 0,
+      forecasts: [],
+      location: {
+        city: 'manchester',
+        country: '',
+      },
+    };
+  }
+
+  goFetchNow = city => {
+    fetch(`${URL}${city}`)
+      .then(data => data.json())
+      .then(data => {
+        this.setState({
+          forecasts: data.forecasts,
+          location: {
+            city: data.location.city,
+            country: data.location.country,
+          },
+        });
+      });
+  };
+
```

Comment:
Well done! Not everybody thought to store the initial city within the app state.